apps/ui/src/Communities/CreateCommunityDialog.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from "@mui/material";
+import React from "react";
+import defaultTheme from "../themes/default";
+import { TEXT } from "../themes/default/consts";
+
+interface CreateCommunityDialogProps {
+    setIsOpen: (isOpen: boolean) => void;
+    isOpen: boolean;
+  }
+
+const CreateCommunityDialog:React.FC<CreateCommunityDialogProps> = ({isOpen, setIsOpen}) => {
+    const handleClickOpen = () => {
+        setIsOpen(true);
+      };
+    
+      const handleClose = () => {
+        setIsOpen(false);
+      };
+
+      const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const formData = new FormData(event.currentTarget);
+        const formJson = Object.fromEntries((formData as any).entries());    
+        const communityName = formJson.communityName;
+        const description = formJson.description;
+        const newCommunity = {
+            name: communityName,
+            description
+        }
+        console.log({newCommunity});
+        
+        //TODO - submit newCommunity
+        handleClose();
+      }
+    
+      return (
+        <React.Fragment>
```

Comment:
this is not needed if you return only the dialog, also it more common to write this as `<></>`